docs/conf.py

```diff
@@ -17,8 +17,8 @@
 
 html_context = {
   'version' : '3.1',
-  'full_version' : '3.1.5',
-  'maven_plugin_version' : '3.1.3.1',
+  'full_version' : '3.1.6',
+  'maven_plugin_version' : '3.1.5',
```

Comment:
Maven plugin version 3.1.5 already exists. Are we releasing 3.1.6?

Reply:
We use the existing latest version at here. After we ship Maven plugin v3.1.6, we can change this part.